mithril-common/src/test_utils/fixture_builder.rs

```diff
@@ -146,25 +144,58 @@ impl MithrilFixtureBuilder {
         }
     }
 
-    fn build_party_with_operational_certificate(
-        &self,
-        party_index: usize,
-        kes_key_seed: &mut [u8],
-    ) -> PartyId {
+    fn provide_kes_key<'a>(
+        key_buffer: &'a mut [u8],
+        kes_key_seed: &'a mut [u8],
+    ) -> (Sum6KesBytes, PublicKey) {
+        if let Some((kes_bytes, kes_verification_key)) =
+            MithrilFixtureBuilder::cached_kes_key(kes_key_seed)
+        {
+            (kes_bytes, kes_verification_key)
+        } else {
+            // TODO We can log a warning to indicate that the cache is not used
```

Comment:
I think it would be a good idea as well.

Reply:
I add a println! to see when this case happens